ci/lint.dart

```diff
@@ -108,20 +140,25 @@ void main(List<String> arguments) async {
   final List<Command> changedFileBuildCommands =
       buildCommands.where((x) => containsAny(x.file, changedFiles)).toList();
 
+  print(_linterOutputHeader);
   int exitCode = 0;
   //TODO(aaclarke): Coalesce this into one call using the `-p` arguement.
   for (Command command in changedFileBuildCommands) {
-    final String tidyArgs = calcTidyArgs(command);
-    final List<String> args = [command.file, checks, '--'];
-    args.addAll(tidyArgs.split(' '));
-    print('# linting ${command.file}');
-    final Process process = await Process.start(tidyPath, args,
-        workingDirectory: command.directory, runInShell: false);
-    process.stdout.transform(utf8.decoder).listen((data) {
-      print(data);
-      exitCode = 1;
-    });
-    await process.exitCode;
+    if (!(await shouldIgnoreFile(command.file))) {
+      final String tidyArgs = calcTidyArgs(command);
+      final List<String> args = [command.file, checks, '--'];
+      args.addAll(tidyArgs.split(' '));
+      print('🔶 linting ${command.file}');
```

Comment:
Maybe not for this PR, but consider adding a --verbose flag, and giving output only if there is an error when --verbose is not passed.

Reply:
Good call, added comment.  I don't want to do it now until we've chased out all the problems with finding the diff'ed files.